lib/tabula.ex

```diff
@@ -61,6 +62,11 @@ defmodule Tabula do
     def keys(row), do: row |> Keyword.keys
   end
 
+  defimpl Row, for: Any do
+    def get(%{__struct__: _} = row, col, default \\ nil), do: row |> Map.get(col, default)
+    def keys(row), do: row |> Map.keys |> tl
```

Comment:
I think maps are unordered so we can't rely on that assumption.
`row |> Map.from_struct |> Map.keys` would be a better fit here.